### ISSUE TITLE
Access to VPC inside deployment docker runtime

### ISSUE THREAD

**Issue by denissabramovs** [aws-amplify/amplify-cli#4516]
Allow AWS Amplify deployment process to access existing default VPC or provide an option as AWS CodeBuild project has here:

https://aws.amazon.com/blogs/devops/access-resources-in-a-vpc-from-aws-codebuild-builds/

We tried to implement sequelize/typeorm database migrations inside AWS Amplify deployment pipeline, but failed to do so, because it's isolated from the existing VPC...

This would be great to have that there.

**Comment by undefobj** [comment]
Can you go into more details on your use case?

**Comment by denissabramovs** [comment]
Yes ofc. We need to do Postgres migrations after all CloudFormation stacks are deployed.
Like this:
```
    build:
      commands:
        - '# Execute Amplify CLI with the helper script'
        - amplifyPush --simple
        - cd "${ROOT_FOLDER}/amplify/backend/function/SomeLambdaFunction/src"
        - npm run migration:run
```

**Comment by denissabramovs** [comment]
`npm run migration:run`  is running `./node_modules/.bin/ts-node ./node_modules/.bin/typeorm migration:run`
This is documented here: https://github.com/typeorm/typeorm/blob/master/docs/migrations.md
This command is reading all database migration scenarios and apply them in the correct order and update state with launched migrations to avoid launching them again in the future.
Sequelize is working the same way.
Both typeorm and sequelize are ORM libraries.
But the problem is, that AWS Amplify docker **container runtime doesn't have access to default VPC** and because of that it can't reach RDS Postgres database to apply migration changes into it and update state of migrations scenarios.

**Comment by denissabramovs** [comment]
We will try to implement that inside CodeBuild/CodePipeline services, because CodeBuild have it built in as described here https://aws.amazon.com/blogs/devops/access-resources-in-a-vpc-from-aws-codebuild-builds/

**Comment by denissabramovs** [comment]
But it would be cool to have it configurable the same was in AWS Amplify.
